grpc/client/client.go

```diff
@@ -527,6 +530,10 @@ func (c *Client) processIncommingServerMessages() {
 func (c *Client) dbWatchdog() {
 	for {
 		select {
+		case <-c.watchDogStopChan:
+			return
```

Comment:
This will prevent re-registering the client after it being stopped.
The watchdog should always work while the agent is running. Otherwise, fix for https://jira.percona.com/browse/PBM-239 (https://github.com/percona/percona-backup-mongodb/commit/8114b8f8471ff19cba6acbb23bd7c8e8ea649c65) won't work.